### PR TITLE
ACL Deployment Customizations writeLBAddressPrefix

### PR DESCRIPTION
**Pull request by ryazhang-microsoft**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

File: specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2024-07-09-preview/examples/ConfidentialLedger_Create.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
         "hostLevel": "Info",
         "maxBodySizeInMb": 1,
         "subjectName": "CN=CCF Node",
-        "nodeCount": 3
+        "nodeCount": 3,
+        "writeLBAddressPrefix": "write"
```
Comment (msftsettiy) [inline specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2024-07-09-preview/examples/ConfidentialLedger_Create.json:28]: Please add a customization for the worker threads and limit it to 2. Default value is 0.

Comment (ryazhang-microsoft) [reply specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2024-07-09-preview/examples/ConfidentialLedger_Create.json]: added

File: specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2024-07-09-preview/examples/ConfidentialLedger_Create.json [hunk 2]
```diff
@@ -24,7 +24,9 @@
         "hostLevel": "Info",
         "maxBodySizeInMb": 1,
         "subjectName": "CN=CCF Node",
-        "nodeCount": 3
+        "nodeCount": 3,
+        "writeLBAddressPrefix": "write",
+        "workerThreads": 1
```
Comment (msftsettiy) [inline specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2024-07-09-preview/examples/ConfidentialLedger_Create.json:29]: Default value is 0

**Review by msftsettiy — APPROVED** [review]

**Pull request merged**